First source code version (#1)
dextra/flutter_bifrost: README.md

# Bifrost

> Bifrost is a plugin which enables hybrid integration of Flutter for your existing native apps.

Reviewer: ```suggestion
Bifrost is a plugin which enables easy embedding of your Flutter application into your existing native apps.
```

## Requirements

- Flutter 1.22.6+ on stable channel
- iOS 10.0+ Xcode 12.0+ Swift 5+
- Android minSdkVersion 16 Kotlin 1.3.50+

## Getting Started

### Flutter

Add a dependency in your Flutter project

``` yaml
dependencies:
  bifrost: ^0.0.1
```

Initialize Bifrost on your MaterialApp or CupertinoApp

``` dart
@override
Widget build(BuildContext context) {
  return MaterialApp(
    builder: Bifrost.init(),
    routes: routes,
    theme: theme,
  );
}
```

> Invoke method on common method channel

Reviewer: ```suggestion
Invoke a method of the native layer, from your Flutter code
```

> # bifrost

Author: OK I will do it.
